.gitmodules

```diff
@@ -168,4 +168,4 @@
 	ignore = untracked
 	path = vendor/negentropy
 	url = https://github.com/waku-org/negentropy.git
-	branch = feat/c-wrapper
\ No newline at end of file
+	branch = chore/optimizations
```

Comment:
I'm a bit confused on the best way to use gitmodule.

Would it not be better to update one branch instead of using multiple branch?

My only experience so far has been pain and suffering. :performing_arts:  :rofl:

Reply:
Yeah, i will switch this to use master. 
Just using branches there in case i want to revert something, but now i can start moving to master.

I agree, My experience setting this sub-module up also has been painful especially with branches.